### PR TITLE
Removed unsupported install methods

### PR DESCRIPTION
**Pull request by rich-howell**
Removing unsupported install methods

### CODE REVIEW

File: docs/Installing/fly/Fly.io.md [hunk 1]
```diff
@@ -15,10 +15,9 @@ step unclear, [contact us](/Contact) for support.
 ## Deployment methods
 
 Fly offers a variety of ways to get an application deployed on their infrastructure. We currently
-have community contributed documentation for the following deplyment methods
+have community contributed documentation for the following deployment methods
 - Deploying from a clone of the main git repository [(link)][git-guide]
 - Deploying from a published container image [(link)][image-guide]
```
Comment (j-f1) [inline docs/Installing/fly/Fly.io.md:20]: Nit: while updating this:
```suggestion
- Deploying [from a clone of the main git repository][git-guide]
- Deploying [from a published Docker container image][image-guide]
```

Comment (rich-howell) [reply docs/Installing/fly/Fly.io.md:20]: Yeah, working on it :)

File: docs/Installing/overview.md [hunk 2]
```diff
@@ -4,14 +4,13 @@ title: 'Installing Actual'
...
 
-While running a server can be a complicated endeavor, we’ve tried to make it fairly easy to set up and hands-off to maintain. Choose one of the following options to get started:
+While running a server can be a complicated endeavour, we’ve tried to make it fairly easy to set up and hands-off to maintain. Choose one of the following options to get started:
 
 - If you’re not comfortable with the command line and are willing to pay a small amount of money to have your version of Actual hosted on the cloud for you, we recommend [PikaPods](PikaPods.md).
 - If you’re willing to run a few commands in the terminal:
   - [Fly.io](fly/Fly.io.md) offers free cloud hosting. (Make sure you read the section about [persisting your data](fly/Fly-persisting.md)!)
   - You could [directly install Actual locally](Local/your-own-machine.md) on macOS, Windows, or Linux if you don’t want to use a tool like Docker. (This method is the best option if you want to contribute to Actual!)
-  - If you want to use Docker, we have instructions for [running Actual directly](Docker.md) inside Docker or [behind an nginx proxy that offers HTTPS support](DockerWithNginx.md).
```
Comment (j-f1) [inline docs/Installing/overview.md:13]: I think we still support Docker, right?

```markdown
  - If you want to use Docker, we have instructions for [using our provided Docker containers](Docker.md).
```

Comment (rich-howell) [reply docs/Installing/overview.md:13]: You're right.

**Pull request merged**